### ISSUE TITLE
Need develop branch to get a better version control

### ISSUE THREAD

**Issue by fujiechen** [yihuashub/yihuapost#4]
It should have `develop` branch to do the version control of development. All new changes should appear as new feature branch and be merged to `develop`. `develop` branch should be merged to `master` when the project need a new stable release.

All new issue should controlled by Issues and it will be solved once the related pull request is merged to develop.

https://www.atlassian.com/git/tutorials/comparing-workflows

**Comment by LongWinter** [comment]
👍
